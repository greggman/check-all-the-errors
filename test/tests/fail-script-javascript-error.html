<script>
console.foo();   // foo does not exist
</script>