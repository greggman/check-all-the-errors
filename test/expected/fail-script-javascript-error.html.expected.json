{
  "numErrors": 1,
  "pages": {
    "http://localhost:8090/fail-script-javascript-error.html": {
      "status": 200,
      "errors": [
        {
          "type": "pageerror",
          "error": "Error: TypeError: console.foo is not a function\n    at http://localhost:8090/fail-script-javascript-error.html:2:9"
        }
      ]
    }
  },
  "responses": [
    {
      "href": "http://localhost:8090/fail-script-javascript-error.html",
      "resourceHref": "http://localhost:8090/fail-script-javascript-error.html",
      "status": 200
    }
  ]
}